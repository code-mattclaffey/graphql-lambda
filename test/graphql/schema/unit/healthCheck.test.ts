import healthCheckSchema from "../../../../src/graphql/schema/healthCheck";

describe("healthCheckSchema", () => {
  it("should return GOOD on the health check query", () => {
    expect(healthCheckSchema).toMatchSnapshot();
  });
});
