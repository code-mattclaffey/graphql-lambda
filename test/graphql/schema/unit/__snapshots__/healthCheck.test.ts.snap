// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`healthCheckSchema should return GOOD on the health check query 1`] = `
{
  "definitions": [
    {
      "description": undefined,
      "directives": [],
      "kind": "EnumTypeDefinition",
      "name": {
        "kind": "Name",
        "value": "HealthStatus",
      },
      "values": [
        {
          "description": undefined,
          "directives": [],
          "kind": "EnumValueDefinition",
          "name": {
            "kind": "Name",
            "value": "GOOD",
          },
        },
        {
          "description": undefined,
          "directives": [],
          "kind": "EnumValueDefinition",
          "name": {
            "kind": "Name",
            "value": "BAD",
          },
        },
      ],
    },
    {
      "description": undefined,
      "directives": [],
      "fields": [
        {
          "arguments": [],
          "description": undefined,
          "directives": [],
          "kind": "FieldDefinition",
          "name": {
            "kind": "Name",
            "value": "health",
          },
          "type": {
            "kind": "NamedType",
            "name": {
              "kind": "Name",
              "value": "HealthStatus",
            },
          },
        },
      ],
      "interfaces": [],
      "kind": "ObjectTypeDefinition",
      "name": {
        "kind": "Name",
        "value": "Query",
      },
    },
  ],
  "kind": "Document",
  "loc": {
    "end": 89,
    "start": 0,
  },
}
`;
